commands/intakeCommand.py

```diff
@@ -0,0 +1,89 @@
+import wpilib
+import commands2
+import commands2.cmd
+
+from subsystems.intakeSubsystem import intakeSubsystem
+
+class intake(commands2.Command):
+    
+    def __init__(self, intakeSubsystem: intakeSubsystem) -> None:
+        super().__init__()
+        self.intakeSub = intakeSubsystem
+
+    def initialize(self):
+        import logging
+        logger = logging.getLogger("jhony")
+        logger.info("IntakeRunning")
+
+    def execute(self):
+        self.intakeSub.intake()
+
+    def isFinished(self):
+        return False
```

Comment:
Instead of having all these commands never finish until interrupted, I think (but this should be tested) that you could have them all end instantly.  Once you set a motor to a value, it stays at that value until set to something else.  You don't need to keep setting the motor value during `execute()`, either directly or by using a method such as `intakeSub.intake()`.  So in this case, you could use the `initialize()` method to call `self.intakeSub.intake()`, and have `isFinished()` return `True`.

This way works, and you don't need to change it, but the other way might be cleaner in the future.